README.md

```diff
@@ -37,8 +37,15 @@ See [the manpages](https://shopify.github.io/ecfg) for more technical documentat
   is `*{json,yaml,yml,toml}`, this flag is optional, but when the type can't
   be inferred because of a different filename or when reading from stdin, it
   must be specified. (note that this recognizes `*.ejson` as json automatically)
-* All references to EJSON have changed to ECFG, of course, including
-  `/opt/ecfg/keys` and `ECFG_KEYDIR`.
+* All references to EJSON have changed to ECFG, including `ECFG_KEYDIR`.
+* Key lookup semantics are a bit more complex: In addition to
```

Comment:
I think this text makes it a little more approachable and readable, without overwhelming the user with the word "complex"